react/frontpage/components/shows.scss

```diff
@@ -0,0 +1,212 @@
+  padding: 5px;
+  cursor: pointer;
+}
+
+.switchStyle {
+  display: inline-block;
+  position: relative;
+  padding: 5px;
+  cursor: pointer;
+}
+
+.schedHeader {
+  display: inline;
+  font-size: 50px;
+  font-weight: 100;
+  font-style: italic;
+  margin-top: -15px;
+}
+
+.scheduleViewButton {
+  display: inline;
+  position: relative;
+  border-radius: 2px;
+  border: 0.5px solid black;
+  height: 50px;
+  left: 30px;
+  top: -8px;
+  padding: 2px;
+}
+
+/* SHOW GRAPH */
+
+.colorKey {
+  display: block;
+  position: relative;
+}
+
+.colorKeyPara {
+  font-size: 13px;
+  position: relative;
+  display: inline-block;
+}
+
+.dotCur {
+  position: relative;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  display: inline-block;
+  margin-left: 10px;
+  margin-right: 20px;
+  background-color: rgba(60, 132, 204, 0.45);
+}
+
+.dotSpot {
+  position: relative;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  display: inline-block;
+  margin-left: 10px;
+  margin-right: 15px;
+  background-color: rgba(128, 0, 128, 0.45);
+}
+
+.headsStyle {
+  font-size: 11px;
+  padding-right: 1px;
+  margin-bottom: 2px;
+  display: inline-block;
+  position: relative;
+}
+
+.timeStyle {
+  font-size: 11px;
+  padding-right: 3px;
+  display: inline-block;
+  position: relative;
+  margin-right: -15px;
+  top: -5px;
+}
+
+.hourBlocks {
+  width: 100%;
+  margin-bottom: -10px;
+  display: inline-block;
+  position: relative;
+}
+
+/* SHOW GRAPH BLOCKS */
+
+.boringBlockStyle {
+  height: 100%;
+  width: 100%;
+  background-color: white;
+  display: inline-block;
+  position: relative;
+}
+
+.blockStyle {
+  cursor: pointer;
+  height: 100%;
+  width: 100%;
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  background: rgba(255, 255, 255, 0.45);
+  height: 100%;
+}
+
+/* SHOW BLURB */
+
+.blurbStyle {
+  width: 40%;
+  float: left;
+  font-size: 12px;
+  font-weight: lighter;
+  margin-top: -50px;
+}
+
+.lineStyle {
+  border-top: 0.5px solid black;
+  height: 0px;
+  width: 100%;
+  position: relative;
+  margin-bottom: 2px;
+  top: -5px;
+}
+
+/* GENERAL */
+
+.header {
+  font-size: 15px;
+  font-weight: lighter;
+}
+
+.djs {
+  font-size: 15px;
+  font-style: italic;
+}
+
+.showTitle {
+  font-size: 20px;
+  font-weight: lighter;
+}
+
+.blurb {
+  font-size: 12px;
+  font-weight: lighter;
+}
+
+.dot {
+  background-color: grey;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  display: inline-block;
+  margin-left: 5px;
+  position: relative;
+}
+
+.Headline {
+  h1 {
+    display: inline;
+    font-size: 50pt;
+  }
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.chart-container {
+  display: block;
+  margin: 0;
+  padding-top: 15px;
+}
+
+.day {
+  display: inline-block;
+  position: relative;
+  width: 8%;
+  h1 {
+    display: inline;
+    font-size: 15pt;
+  }
+}
+
+.showBlock {
+  opacity: 0.5;
+  height: 20px;
+  padding-right: 5px;
+  display: inline-block;
+  position: relative;
+}
```

Comment:
⚠️ Once it's cleaned up a bit, I'd like someone else to look at the css styles and see if we can clean it up at all, especially with SASS capabilities